chunkflow/flow.py

```diff
@@ -336,6 +341,47 @@ def normalize_section_shang_cmd(tasks, name, nominalmin, nominalmax, clipvalues)
             task['log']['timer'][name] = time() - start
         yield task
 
+@cli.command('arbitrary-chunk-op')
```

Comment:
good idea to plugin more operators!
how about renaming it to `custom-operator`?